packages/odyssey/src/scss/components/_card.scss

```diff
@@ -56,12 +56,12 @@
     padding-bottom: $base-spacing;
   }
 
-  .card--state + & {
+  .ods-card--state + & {
     padding-top: $base-spacing;
```

Comment:
Probably related to my other question, but do we want to prefix variables as well? If not, why?

Reply:
For above as well - I only prefixed things that would end up compiled to avoid collisions in production.

Doing the same for variables, mixins, etc might be smart too